README.md

```diff
@@ -1,57 +1,58 @@
-# A template repo for IIASA Python projects
+# Documentation of the teaching learning material by the Energy, Climate, and Environment Program (ECE)
 
 Copyright (c) 2021 IIASA
 
-![License](https://img.shields.io/github/license/iiasa/python-stub)
-[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
+![License](https://img.shields.io/github/license/iiasa/ece-teaching)
 
-## Overview
+This is a [Sphinx](http://sphinx-doc.org/) project for the documentation of the build of the teaching and learning website by the Energy, Climate, and Environment Program (ECE). The website can be found [here](https://teaching.ece.iiasa.ac.at/).
 
-Template repository for creating python packages and Sphinx-based documentation pages in line with the IIASA design guidelines
+## Scope & Aim
 
-## Configuration
+The scope and aim of the repository is to create a Sphinx documentation, which is the basis for the teaching and learning website. This repository should contain all docs, but not the data, of the teaching and learning material. 
 
-To start a new Python package from this repo, 
-click on the green **Use this template** button on top-right of this page.
-Detailed instructions to create a new repository from a template can be found
-[here](https://help.github.com/en/articles/creating-a-repository-from-a-template).
+## Instructions
 
-Then, make the following changes:
+### How to install dependencies and build locally
+1. Install dependencies and start Sphinx [sphinx_rtd_theme](https://sphinx-rtd-theme.readthedocs.io/)
```

Comment:
Why is there a link to the rtd-theme? If you want to mention it, maybe write a full sentence like: "The teaching pages use the Sphinx-RTD-Theme customized to follow the IIASA style & design guidelines.

Reply:
Ah sorry, yes this link is not the correct one.